userspace/libsinsp/sinsp.cpp

```diff
@@ -815,19 +815,23 @@ void sinsp::on_new_entry_from_proc(void* context,
 	//
 	if(fdinfo == NULL)
 	{
+		bool thread_added = false;
 		sinsp_threadinfo* newti = new sinsp_threadinfo(this);
```

Comment:
I'd rather that we put the thread a unique_ptr then call release() when it is added to the thread manager.

But I am fine with this.